compiler/ballerina-lang/src/main/resources/grammar/BallerinaParser.g4

```diff
@@ -98,12 +98,12 @@ fieldDefinition
     ;
 
 recordRestFieldDefinition
-    :   typeName restDescriptorPredicate ELLIPSIS
+    :   typeName restDescriptorPredicate ELLIPSIS SEMICOLON
     |   sealedLiteral
     ;
 
 sealedLiteral
-    :   NOT restDescriptorPredicate ELLIPSIS
+    :   NOT restDescriptorPredicate ELLIPSIS SEMICOLON
```

Comment:
`sealedLiteral` has been used in places like `arrayTypeNameLabel` and `restBindingPattern`. Won't this change affect them?
Example of an array with fixed length: 
```ballerina
int[!...] x = [1, 2];
int[!...;] x = [1, 2];
```

Reply:
Yes, it is affecting. This PR is still WIP :) I did not send any PR review request yet :)

Anyway, thanks for pointing out.

Reply:
Oh sorry didn't know it was WIP :D